content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If your Kubernetes cluster is to run critical workloads, it must be configured to be resilient.
+
+<!-- body -->
+
+## Production issues
+
+Typically, a production environment Kubernetes has more requirements than a
```

Comment:
nit: ... a production environment Kubernetes cluster? OR
a production Kubernetes cluster environment?

Reply:
Yeah, that could be better. I'll use your second suggestion.